scripts/test/test-utils.sh

```diff
@@ -76,16 +77,16 @@ run_tests() {
 
   # Need to decompress images built with MENDER_COMPRESS_DISK_IMAGE=gzip before
   # running tests.
-  if [ -f deploy/${device_type}-${artifact_name}-mender.img.gz ]; then
+  if [ -f deploy/*.gz ]; then
     # sudo is needed because the image is created using docker-mender-convert
     # which sets root permissions on the image
-    sudo gunzip --force deploy/${device_type}-${artifact_name}-mender.img.gz
+    sudo gunzip --force deploy/*.gz
```

Comment:
Why `*.gz`? This can easily break if you have more than one image in the directory while testing locally. This should be `${converted_image_name}.gz`